src/QmlControls/HorizontalFactValueGrid.qml

```diff
@@ -126,7 +121,7 @@ T.HorizontalFactValueGrid {
                     Layout.preferredHeight: ScreenTools.minTouchPixels
                     Layout.preferredWidth:  parent.width
                     text:                   qsTr("+")
-                    onClicked:              appendColumn()
```

Comment:
This check should be on the enabled of the button, or in the parent that shows such buttons. Otherwise the user will click the button and nothing will happen.